torch/nn/modules/loss.py

```diff
@@ -151,9 +151,15 @@ class NLLLoss(_WeightedLoss):
         - Input: :math:`(N, C)` where `C = number of classes`, or
             :math:`(N, C, d_1, d_2, ..., d_K)` with :math:`K \geq 2`
             in the case of `K`-dimensional loss.
+            Eg:  [0,0,1]
```

Comment:
The inputs don't have to be one-hot coded. If you want to show an example of the inputs, I think it would be better to put this into the examples section below.